admin_guide/allocating_node_resources.adoc

```diff
@@ -56,9 +56,14 @@ kubeletArguments:
 ----
 ====
 
-Currently, the `*cpu*` and `*memory*` resource types are supported. For `*cpu*`,
+Currently, the `*cpu*`, `*memory*`, and, if you have enabled the ephemeral storage tech preview in {product-title} 3.10, `*ephemeral-storage*` resource types are supported. For `*cpu*`,
```

Comment:
nit: reads awkwardly.

The `cpu`, and `memory` resource types are fully supported.  If the ephemeral storage tech preview feature is enabled, the `ephemeral-storage` resource type is available as well.

Reply:
+1